docs/guides/migrations/launchdarily-migration.md

```diff
@@ -136,7 +136,9 @@
           return assignment;
         }
         ```
-        
+        If you wnat to use JSON or Numeric variants, you will have
```

Comment:
typo `wnat`

Reply:
cc @bertilhatt

Reply:
Thank you!

Reply:
Fixed in https://github.com/Eppo-exp/eppo-docs/pull/461